lib/datadog/notifications/config.rb

```diff
@@ -20,7 +20,7 @@ def use(klass, opts = {})
       def connect!
         env = ENV['RAILS_ENV'] || ENV['RACK_ENV']
         tags.push("env:#{env}")       if env && tags.none? {|t| t =~ /^env\:/ }
-        tags.push("host:#{hostname}") if tags.none? {|t| t =~ /^host\:/ }
+        tags.push("host:#{hostname}") if tags.none? {|t| t =~ /^host\:/ } && hostname
```

Comment:
i would just turn this around, e.g. `tags.push("host:#{hostname}") if hostname && tags.none? {|t| t =~ /^host\:/ }`